__test__/integration/debt_token_api/runners/transfer_from.ts

```diff
@@ -134,9 +148,15 @@ export class TransferFromScenarioRunner extends ScenarioRunner {
                 });
             } else {
                 test(`throws ${scenario.errorType}`, async () => {
-                    await expect(transferFromPromise).rejects.toThrow(scenario.errorMessage);
+                    await expect(apiCallPromise).rejects.toThrow(scenario.errorMessage);
                 });
             }
         });
     }
+
+    protected getAPICallPromise(params: TransferAPICallParameters): Promise<string> {
```

Comment:
we should define this method signature in an interface that the `transfer` and `transferFrom` runners implement.